vault/resource_cert_auth_backend_role_test.go

```diff
@@ -118,6 +120,49 @@ func TestCertAuthBackend(t *testing.T) {
 	})
 }
 
+func TestCertAuthBackend_OCSP(t *testing.T) {
+	backend := acctest.RandomWithPrefix("tf-test-cert-auth")
+	name := acctest.RandomWithPrefix("tf-test-cert-name")
+
+	resourceName := "vault_cert_auth_backend_role.test"
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testutil.TestAccPreCheck(t)
+			SkipIfAPIVersionLT(t, testProvider.Meta(), provider.VaultVersion113)
+		},
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testCertAuthBackendDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testCertAuthBackendConfig_OCSP_default(backend, name),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "backend", backend),
+					resource.TestCheckResourceAttr(resourceName, "name", name),
+					resource.TestCheckResourceAttr(resourceName, fieldOCSPServersOverride+".#", "0"),
+					resource.TestCheckResourceAttr(resourceName, fieldOCSPCACertificates, ""),
+					resource.TestCheckResourceAttr(resourceName, fieldOCSPEnabled, "false"),
+					resource.TestCheckResourceAttr(resourceName, fieldOCSPFailOpen, "false"),
+					resource.TestCheckResourceAttr(resourceName, fieldOCSPQueryAllServers, "false"),
+				),
+			},
+			{
```

Comment:
Should we also add an import test here?

Reply:
Looks like import is not supported as we don't have Importer set on the schema.Resource. When I try to add it I get a failure. I think we should handle supporting import in a separate PR.

Reply:
Thanks for the context, that makes sense 👍🏼